utilities/generate_messages_catalog.py

```diff
@@ -97,15 +100,15 @@ def _log_function(item):
 
 
 FUNC_HANDLER = {
-    "info": ("info", 0),
-    "warning": ("warning", 0),
-    "error": ("error", 0),
-    "exception": ("exception", 0),
+    "info": lambda x: ("info", 0),
+    "warning": lambda x: ("warning", 0),
+    "error": lambda x: ("error", 0),
+    "exception": lambda x: ("exception", 0),
```

Comment:
Certainly. Switching these to lambda functions was to reduce complexity in a loop further down. With the exception of the dynamic level, all others return a tuple of themselves and a log level of 0. Rather than checking each time through the loop of the elements in the walk for if something was a function or tuple, Andrew's suggestion was to just lambda them all out and always call it as a function. It reduces the complexity of the already complex loop function as well as making it more readable.